crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownProceedingControllerTest.java

```diff
@@ -131,9 +131,9 @@ void updateApplication_BadRequest_RequestEmptyBody() throws Exception {
 
     @Test
     void givenAValidInput_whenUpdateIsInvoked_thenSuccess() throws Exception {
```

Comment:
Test names are a bit unclear, can we update these to reflect we're targeting the `updateCrownCourt` endpoint?